### DISCUSSION TITLE
topic detection missing details and documentation.

### DISCUSSION THREAD

**Discussion by ShaiStudny** [deepgram/community#504] [General help] [answered]
Hi,
I'm looking for additional details regarding "Topic Detection" in order to implement and cannot find these in the documentation. 

https://developers.deepgram.com/docs/topic-detection#identifiable-topics/

1,
Is there a predefined full list of all the possible "identifiable topics" in the Topic Detection responses?
Is this a list of topics a specific built by deepgram (and if so, where can it be found?), or is the full list one of the public IAB taxonomies (and if so, which version)?

2,
What is the coverage of Topic Detection in terms of languages?
Is it applied to English transcripts only, or additional languages are also supported?

Thanks

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment, marked as answer]
Hi @ShaiStudny, rather than using a predefined list of possible topics, Deepgram detects the topics on-the-fly and returns relevant info. The topic categories listed in the documentation you linked are the possible categories that can be detected. This list will be updated as additional categories are added.

Topic detection currently only supports English. As we add support for additional languages, the docs will be updated. For future reference, if you look at the top of the page you linked you'll see "English (all available regions)" in a pink box. This indicates which languages are supported.
